pkg/collector/corechecks/system/cpu.go

```diff
@@ -39,6 +43,13 @@ func (c *CPUCheck) Run() error {
 		return err
 	}
 
+	ctxSwitches, err := c.readCtxSwitches()
+	if err != nil {
+		log.Errorf("system.CPUCheck could read context switches: %s", err.Error())
```

Comment:
Since this is a "new" metric, if it fails to be collected, I think it's better to avoid spamming the logs (this would be logged every 15 seconds otherwise) even if that means users will have to move to debug level to have more info on why it failed to be collected.